src/render/painter.js

```diff
@@ -400,24 +454,30 @@ class Painter {
         }
 
         gl.linkProgram(program);
-        assert(gl.getProgramParameter(program, gl.LINK_STATUS), gl.getProgramInfoLog(program));
+        assert(gl.getProgramParameter(program, gl.LINK_STATUS), (gl.getProgramInfoLog(program): any));
 
         const numAttributes = gl.getProgramParameter(program, gl.ACTIVE_ATTRIBUTES);
         const result = {program, numAttributes};
 
         for (let i = 0; i < numAttributes; i++) {
             const attribute = gl.getActiveAttrib(program, i);
-            result[attribute.name] = gl.getAttribLocation(program, attribute.name);
+            if (attribute) {
+                result[attribute.name] = gl.getAttribLocation(program, attribute.name);
+            }
```

Comment:
This doesn't work, unfortunately:

```
src/render/painter.js:465
465:             result[attribute.name] = gl.getAttribLocation(program, attribute.name);
                                  ^^^^ property `name`. Property cannot be accessed on possibly null value
465:             result[attribute.name] = gl.getAttribLocation(program, attribute.name);
                        ^^^^^^^^^ null
```

Flow doesn't know to treat `assert(false)` as diverging.